.github/fabricbot.json

```
@@ -3747,6 +3703,45 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "labelAdded",
+              "parameters": {
+                "label": "help wanted"
+              }
+            },
+            {
+              "name": "isAction",
+              "parameters": {
+                "action": "labeled"
+              }
+            }
+          ]
+        },
+        "eventType": "issue",
+        "eventNames": [
+          "issues",
+          "project_card"
+        ],
+        "taskName": "Drop a comment when a help candidate issue is identified",
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Looks liek this issue has been identified as a candidate for community contribution. If you're a community member, considering to send a PR for this issue, look for the `Summary Comment` link in the issue description. That comment has been left by an engineer on our team to help you get started with handling this issue. It contains high level guidance about what what approach you can take to address this issue as well as references to some useful relevant material. Good luck with your endeavour and we're looking forward to your PR. You can learn more about our Help Wanted process [here](https://aka.ms/aspnet/processes/help-wanted)"
```

> If you're a community member, considering to send a PR for this issue

Does "community member" include potential first-time-contributors?

I interpret this as:
* community member -> has already contributes in any form
* so only people that have contributed are "allowed" to work on that issue

So maybe just drop that:
```suggestion
              "comment": "Looks liek this issue has been identified as a candidate for community contribution. If you're considering to send a PR for this issue, look for the `Summary Comment` link in the issue description. That comment has been left by an engineer on our team to help you get started with handling this issue. It contains high level guidance about what what approach you can take to address this issue as well as references to some useful relevant material. Good luck with your endeavour and we're looking forward to your PR. You can learn more about our Help Wanted process [here](https://aka.ms/aspnet/processes/help-wanted)"
```
makes it also a bit shorter and makes it more likely to be actually read.